source/_integrations/isy994.markdown

```diff
@@ -330,13 +330,19 @@ A device is created by creating a directory, with the name for the device, under
 - *HA.lock* will create a lock.
 - *HA.switch* will create a switch.
 
-A program, named *status*, is required under the program device directory. A program, named *actions*, is required for all program devices except for binary_sensor. Any other programs in these device directories will be ignored. The *status* program requires that you create a variable with the name of your choice. This variable will store the actual status of the new device and will be updated by the *action* program.
+A program, named *status*, is required under the program device directory. A program, named *actions*, is required for all program devices except for binary_sensor. Any other programs in these device directories will be ignored.
 
 <p class='img'>
   <img src='/images/isy994/isy994_CoverExample.png' />
 </p>
 
-The *status* program in this directory is what indicates the state of the device:
+The *status* program requires that you create a state variable with the name of your choice. Note that you must use a state variable, not an integer variable. This variable will store the actual status of the new device and will be updated by the *action* program.
+
+<p class='img'>
+  <img src='/images/isy994/isy994_SwitchVariableExample.png' />
```

Comment:
- Please lower care the file name and use snake case filename styling.
- Please use Markdown to embed the image.